docs/faq.rst

```diff
@@ -67,7 +67,9 @@ and 2) whether it is **committed** to the device or not (the data is sometimes
 referred to as being *sticky* to the device).
 
 By default, JAX arrays are placed uncommitted on the default device
-(``jax.devices()[0]``).
+(``jax.devices()[0]``), which is the first GPU by default. The default
```

Comment:
If no GPU is present, `jax.devices()[0]` is the first CPU.

Reply:
Done!